omniledger/service/service.go

```diff
@@ -474,12 +480,28 @@ func (s *Service) verifySkipBlock(newID []byte, newSB *skipchain.SkipBlock) (val
 	txs := data.Transactions
 	cdb := s.getCollection(newSB.Hash)
 	for _, tx := range txs {
-		f, ok := s.verifiers[string(tx.Kind)]
-		if ok {
-			validSB = validSB && f(cdb, &tx)
+		f, exists := s.verifiers[string(tx.Kind)]
+		if !exists {
```

Comment:
Question: what to do when we don't know the verifier function?
1. return `false`, as we suppose the block is invalid
2. ignore, because other nodes might know this function and verify it
I'm leaning towards 1., but 2. has also it's merits. What do you think?

Reply:
I prefer 1. as well, I believe 2. would let the leader introduce arbitrary data of unknown kind into the collection.
Also, in 1., if only a small fraction of nodes doesn't know the function, the block can still pass provided enough votes were gathered.